pages/reference/OS/updates/update-process.md

```diff
@@ -3,32 +3,101 @@ title: Update process details
 
-Rollbacks is a framework designed to automatically roll back the OS update in case something goes wrong.
+__Note:__ Once a host OS update is successful, it is not possible to roll back to a previous OS version (except via [automatic rollbacks](#automatic-rollbacks) as noted below).
 
+#### Automatic Rollbacks
+For failures related to the boot partition, the latest versions of {{ $names.os.lower }} have a rollback feature that will leave the partition in a good state.
 There are two rollback mechanisms in the OS, covering different update failure modes. One based on health checks called the `rollback-health`, and the recognizing if the new system is unbootable for some reason, the `rollback-altboot`. Learn more about [rollback mechanisms][rollback-mechanism].
 
+#### Delta Updates and Bandwidth Efficiency
+A delta update contains only the differences between the current OS and the new version. This reduces the amount of data that needs to be transferred and can significantly lower network costs and update times. 
+
+
+### Logging
+A status of `OS update failed` means the user application should still be running normally, and the reasons for failure can be examined through the update logs at `/mnt/data/balenahup/` (or potentially the legacy location `/mnt/data/resinhup/`). The device may have some reduced functionality, for instance if the supervisor was stopped for the update, but we are working on ways to automatically restore full functionality whenever possible.
+
+## Advanced HUP
+Advanced HUP is only required for certain upgrade paths. The underlying reasons for incompatibility with standard HUP can be, but not limited to:
+
+1. Partition layout changes from one OS version to another. For example:
+    
+    -  The partition layout for NVIDIA JetPack 4-based {{ $names.os.lower }} differs from JetPack 5-based {{ $names.os.lower }}. If a device is running already running JetPack-4 {{ $names.os.lower }}, you can use advanced HUP to update to a JetPack 5-based {{ $names.os.lower }}.
+    - {{ $names.company.upper }} releases a new version of an OS for a certain device type whereby the root partition has been increased to accommodate more kernel module and drivers.
+    -  {{ $names.company.upper }} releases a new version of an OS for a certain device type to include new functionality such as Secure Boot.
+2. Changing OS architecture, for example: 
+    - Switching from a the 32-bit OS variant running on Raspberry Pi 3 to a 64-bit variant of {{ $names.os.lower }}.
+
+__Note:__ If you've used advanced HUP, it does not necessarily mean that future updates won't be compatible with standard HUP. 
+
+### Caveats
+When using the advanced HUP, it’s important to be aware of several limitations compared to the standard HUP mechanism.
+
+- Data loss
```

Comment:
I don't see any special formatting for these bullet points.

![image](https://github.com/user-attachments/assets/f483b889-cafa-43f3-88bd-b716876785a3)